cmd/libasciidoc/main.go

```diff
@@ -23,8 +23,8 @@ func main() {
 	}
 }
 
-var helpTemplate = `
-{{if or .Runnable .HasSubCommands}}{{.UsageString}}{{end}}`
+// var helpTemplate = `
```

Comment:
let's just remove it completely for now, we don't need it and it's taken from an spf13/viper example anyways, so it shouldn't be hard to find again in the future